evennia/commands/default/tests.py

```diff
@@ -543,7 +543,7 @@ def test_new_task_waiting_input(self):
         self.call(system.CmdTasks(), f"/cancel {self.task.get_id()}")
         self.task_handler.clock.advance(self.timedelay + 1)
         self.assertFalse(self.task.exists())
-        self.task = self.task_handler.add(self.timedelay, func_test_cmd_tasks)
+        self.task = self.task_handler.add(self.timedelay + 1, func_test_cmd_tasks)
```

Comment:
This is so arcane that we need a comment here for sure :D 

I think we can also do an explicit check for the os (you can use `if utils.host_os_is('nt')` to set this, since this appears to only be a thing for Windows.